src/charts/css/charts.css

```
@@ -618,6 +618,165 @@ select, input, textarea {
     word-break: break-word;
 }
 
+/* Fenced code / custom blocks: use div instead of <pre> so block UI (e.g. Mermaid) is valid (pre only allows phrasing content). */
+.mdv-textbox-fenced {
+    margin: 0.75em 0;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)